tests/puppeteer/campaigns/smoke/crawlingBO.js

```diff
@@ -0,0 +1,65 @@
+// importing pages
+const LOGIN_PAGE = require('../../pages/BO/login');
```

Comment:
```suggestion
const LoginPage = require('../../pages/BO/login');
```
We need to change all of this. This variable represents the class, therefore it needs to be in this format : `LoginPage`.